### PR TITLE
LPS-161165 Dev Feature Flag removal | Version history in Info Panel

### PR DESCRIPTION
**Pull request by jkappler**
## Motivation

[Link to task](https://issues.liferay.com/browse/LPS-161165)

This pull request removes the FF for Epic related to showing versions history tab in content dashboard, and hide the actions under its corresponding FF

## Change summary:

* Remove FF for versions history tab
* Adds FF for version actions
* Remove unused view

## How to test

* Start the portal without LPS-161013 activated, and the version history should be shown in content dashboard when selecting an item

### CODE REVIEW

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Ei @jkappler , looks very good. I just made a small comment and I request some changes in the frontend tests (I can fix them) because they arte failing.

**Review by beltranrengifo — APPROVED** [review]
LGTM!

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const SidebarHeader = ({actionsSlot, children, className, title}) => {
 			className={classNames(
 				'sidebar-header',
 				{
-					'is-sticky': Liferay.FeatureFlags['LPS-161013'],
+					'is-sticky': true,
```
[the remark this answers is not in the document]
Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js:39]: @claraizquierdo , could you push the changes yourself?

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
thanks @claraizquierdo ❤️

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Review by claraizquierdo — APPROVED** [review]

**Comment by claraizquierdo** [comment]
ci:stop

**Comment by beltranrengifo** [comment]
hi @claraizquierdo !
Is this ready to fw?

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
> hi @claraizquierdo ! Is this ready to fw?

Yes! Just did it. The CI looked stacked.

**Comment by beltranrengifo** [comment]
ci:forward

**Comment by beltranrengifo** [comment]
Hi @claraizquierdo ! it seems the web:packageRunTest have failed... you checked the FE units were green, right?

**Comment by claraizquierdo** [comment]
> 

Hum, yes, I did it. Let me check again 🤔

**Comment by claraizquierdo** [comment]
> > 
> 
> Hum, yes, I did it. Let me check again 🤔

Looks like I forgot to upload the snapshots update 🤦‍♀️  Forwarding again

**Comment by claraizquierdo** [comment]
ci:forward

**Pull request closed without merging**